src/model/transactions.js

```diff
@@ -1,4 +1,4 @@
-import { Schema, ObjectId } from 'mongoose'
+import { Schema, ObjectId, connection } from 'mongoose'
```

Comment:
@MattyJ007 It looks like `connection` is not used anywhere. Is it needed for this module?

Reply:
No :( How embarrassing. I've removed it in https://github.com/jembi/openhim-core-js/pull/1093/commits/adc1b97f4a2f6f5ad7b9cfb695938e2c766acfdd